Scripts/PackageContent.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```

Comment:
wooow, what is that and what does it do? I personally won't be able to support such script as I have no idea what and how it does...